code/game/objects/items/storage/fancy.dm

```diff
@@ -299,6 +299,15 @@
 	var/light_chance = 70 //how likely you are to light the match on the book
 	var/burn_chance = 20 //how likely you are to burn yourself once you light it
 
+/obj/item/storage/fancy/cigar/matchbook/get_examine_text(mob/user)
```

Comment:
why are you duplicating `/obj/item/storage/fancy/get_examine_text(mob/user)` ?

Reply:
because i was thinking the second will override the first when it is used....
but it's not.
help.